ccmlib/scylla_repository.py

```diff
@@ -303,6 +311,32 @@ def setup(version, verbose=True, skip_downloads=False):
     if skip_downloads:
         return directory_name(version), packages
 
+    if validate_by_hash and packages:
+        """
+        Validate if packages hash was changed and the new package(s) have to be downloaded 
+        """
```

Comment:
nit, i'd suggest use a comment instead of a multi-line string for comment. as it cannot be used as doc-string.

Reply:
yep, if we had a linter of any kind on this project, it would have complain about this

Reply:
done